ts/routes/graphs/TrueRetention.svelte

```diff
@@ -4,36 +4,88 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
 -->
 <script lang="ts">
     import type { GraphsResponse } from "@generated/anki/stats_pb";
-    import * as tr from "@generated/ftl";
+    import * as t9n from "@generated/ftl";
```

Comment:
The rest of the codebase uses `tr`, so I think consistency would be better. Would you mind renaming it in this file and the others?

Reply:
Ok. I thought you might ask that.

I did it because I was doing stuff with tables and wanted eliminate confusion between **TR**anslation and **T**able **R**ow